.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
do you feel strongly about `semi: false`? I prefer to not use semicolons, but i honestly don't care that much so whatever you think

Reply:
I have always preferred semi colons mandated. Shall we go with it? Bigger reason is the upstream. I would really not want new contributors spend any more time with these things.